### PR TITLE
update readme for >=24.3

### LINKED ISSUE #95: [LabKey Cloud] Dockerfile: update readme
it contains lots of out of date information, and doesn't have anything regarding tomcat10 updates, or changes end users might need to make.

### PR DESCRIPTION
**Pull request by labkey-willm**
#### Rationale
the README is very out of date compared to recent changes for version 24.3 and beyond. This update fixes that.

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,12 +249,10 @@ Since java can be picky about the position of CLI values, `JAVA_PRE_JAR_EXTRA` a
 
 ## Development Notes
 
-In contrast to `application.properties`, the "startup properties" files housed in `startup/`, are LabKey's own implementation of `.properties` file(s) and generally are less feature rich that Springs'. Environment Variable substitution for example does not function within LabKey `.properties` files, which is why `gettext` is required for `entrypoint.sh`'s use of `envsubst`.
+In contrast to `application.properties`, the "startup properties" files housed in `startup/`, are LabKey's own implementation of `.properties` file(s) and generally are less feature rich that Springs'.
```
Comment (labkey-stuartm) [inline README.md]: typo - should read as 
less feature rich than Springs...

Comment (labkey-stuartm) [reply README.md]: and are specific to LabKey application settings.

**Review by labkey-stuartm — APPROVED** [review]
minor typo you may want to fix

File: README.md (hunk 1 above)
Comment (labkey-willm) [reply README.md]: done

**Pull request merged**